### ISSUE TITLE
anvi'o

### ISSUE THREAD

**Issue by yolanda-huang** [ubccr/software-layer#186]
https://merenlab.org/2015/05/01/installation-v1/#installation-for-developers-youre-a-wizard-arry

**Comment by tonykew** [comment]
Anvi'o version 7.1 test build complete - more info to follow

**Comment by tonykew** [comment]
Anvi'o version 7.1 has been published to the ccrsoft/2023.01 software release


```
vortex1$ module spider anvio
----------------------------------------------------------------------------
  anvio:
----------------------------------------------------------------------------
     Versions:
        anvio/7.1-Python-3.9.6
        anvio/7.1 (E)

Names marked by a trailing (E) are extensions provided by another module.
[...]
vortex1$ module spider anvio/7.1-Python-3.9.6
----------------------------------------------------------------------------
  anvio: anvio/7.1-Python-3.9.6
----------------------------------------------------------------------------
    Description:
      An analysis and visualization platform for 'omics data.


    You will need to load all module(s) on any one of the lines below before the "anvio/7.1-Python-3.9.6" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
[...]
vortex1$ module load gcc/11.2.0 openmpi/4.1.1 anvio/7.1-Python-3.9.6
vortex14 

```

There are very many binaries which are part of this Anvio install:

```
vortex1$ ls -F ${EBROOTANVIO}/bin
00_README.txt*
anvi-analyze-synteny*
anvi-cluster-contigs*
anvi-compute-ani*
anvi-compute-completeness*
anvi-compute-functional-enrichment*
anvi-compute-functional-enrichment-across-genomes*
anvi-compute-functional-enrichment-in-pan*
anvi-compute-gene-cluster-homogeneity*
anvi-compute-genome-similarity*
anvi-compute-metabolic-enrichment*
anvi-db-info*
anvi-delete-collection*
anvi-delete-functions*
anvi-delete-hmms*
anvi-delete-misc-data*
anvi-delete-state*
anvi-dereplicate-genomes*
anvi-display-contigs-stats*
anvi-display-functions*
anvi-display-metabolism*
anvi-display-pan*
anvi-display-structure*
anvi-estimate-genome-completeness*
anvi-estimate-genome-taxonomy*
anvi-estimate-metabolism*
anvi-estimate-scg-taxonomy*
anvi-estimate-trna-taxonomy*
anvi-experimental-organization*
anvi-export-collection*
anvi-export-contigs*
anvi-export-functions*
anvi-export-gene-calls*
anvi-export-gene-coverage-and-detection*
anvi-export-items-order*
anvi-export-locus*
anvi-export-misc-data*
anvi-export-splits-and-coverages*
anvi-export-splits-taxonomy*
anvi-export-state*
anvi-export-structures*
anvi-export-table*
anvi-gen-contigs-database*
anvi-gen-fixation-index-matrix*
anvi-gen-gene-consensus-sequences*
anvi-gen-gene-level-stats-databases*
anvi-gen-genomes-storage*
anvi-gen-network*
anvi-gen-phylogenomic-tree*
anvi-gen-structure-database*
anvi-gen-variability-matrix*
anvi-gen-variability-network*
anvi-gen-variability-profile*
anvi-get-aa-counts*
anvi-get-codon-frequencies*
anvi-get-pn-ps-ratio*
anvi-get-sequences-for-gene-calls*
anvi-get-sequences-for-gene-clusters*
anvi-get-sequences-for-hmm-hits*
anvi-get-short-reads-from-bam*
anvi-get-short-reads-mapping-to-a-gene*
anvi-get-split-coverages*
anvi-get-tlen-dist-from-bam*
anvi-help*
anvi-import-collection*
anvi-import-functions*
anvi-import-items-order*
anvi-import-misc-data*
anvi-import-state*
anvi-import-taxonomy-for-genes*
anvi-import-taxonomy-for-layers*
anvi-init-bam*
anvi-inspect*
anvi-interactive*
anvi-matrix-to-newick*
anvi-mcg-classifier*
anvi-merge*
anvi-merge-bins*
anvi-merge-trnaseq*
anvi-meta-pan-genome*
anvi-migrate*
anvi-oligotype-linkmers*
anvi-pan-genome*
anvi-plot-trnaseq*
anvi-profile*
anvi-profile-blitz*
anvi-push*
anvi-refine*
anvi-rename-bins*
anvi-report-inversions*
anvi-report-linkmers*
anvi-run-hmms*
anvi-run-interacdome*
anvi-run-kegg-kofams*
anvi-run-ncbi-cogs*
anvi-run-pfams*
anvi-run-scg-taxonomy*
anvi-run-trna-taxonomy*
anvi-run-workflow*
anvi-scan-trnas*
anvi-script-add-default-collection*
anvi-script-augustus-output-to-external-gene-calls*
anvi-script-checkm-tree-to-interactive*
anvi-script-compute-ani-for-fasta*
anvi-script-compute-bayesian-pan-core*
anvi-script-enrichment-stats*
anvi-script-estimate-genome-size*
anvi-script-filter-fasta-by-blast*
anvi-script-filter-hmm-hits-table*
anvi-script-fix-homopolymer-indels*
anvi-script-gen-CPR-classifier*
anvi-script-gen-distribution-of-genes-in-a-bin*
anvi-script-gen-functions-per-group-stats-output*
anvi-script-gen-genomes-file*
anvi-script-gen-help-pages*
anvi-script-gen-hmm-hits-matrix-across-genomes*
anvi-script-gen-programs-network*
anvi-script-gen-programs-vignette*
anvi-script-gen-pseudo-paired-reads-from-fastq*
anvi-script-gen-scg-domain-classifier*
anvi-script-gen-short-reads*
anvi-script-gen_stats_for_single_copy_genes.py*
anvi-script-gen_stats_for_single_copy_genes.R*
anvi-script-gen_stats_for_single_copy_genes.sh*
anvi-script-get-collection-info*
anvi-script-get-coverage-from-bam*
anvi-script-get-hmm-hits-per-gene-call*
anvi-script-get-primer-matches*
anvi-script-merge-collections*
anvi-script-permute-trnaseq-seeds*
anvi-script-pfam-accessions-to-hmms-directory*
anvi-script-predict-CPR-genomes*
anvi-script-process-genbank*
anvi-script-process-genbank-metadata*
anvi-script-reformat-fasta*
anvi-script-run-eggnog-mapper*
anvi-script-snvs-to-interactive*
anvi-script-tabulate*
anvi-script-transpose-matrix*
anvi-script-variability-to-vcf*
anvi-script-visualize-split-coverages*
anvi-search-functions*
anvi-search-palindromes*
anvi-search-sequence-motifs*
anvi-self-test*
anvi-setup-interacdome*
anvi-setup-kegg-kofams*
anvi-setup-ncbi-cogs*
anvi-setup-pdb-database*
anvi-setup-pfams*
anvi-setup-scg-taxonomy*
anvi-setup-trna-taxonomy*
anvi-show-collections-and-bins*
anvi-show-misc-data*
anvi-split*
anvi-summarize*
anvi-summarize-blitz*
anvi-tabulate-trnaseq*
anvi-trnaseq*
anvi-update-db-description*
anvi-update-structure-database*
anvi-upgrade*
average_nucleotide_identity.py*
bottle.py*
calc-prorate*
cheroot*
cherryd*
delta_filter_wrapper.py*
django-admin*
django-admin.py*
ete3*
genbank_get_genomes_by_taxon.py*
iu-compute-qual-dicts-from-fastq*
iu-deinterleave-fastq*
iu-demultiplex*
iu-fasta-to-fastq*
iu-fastq-to-fasta*
iu-filter-merged-reads*
iu-filter-quality-bokulich*
iu-filter-quality-minoche*
iu-gen-configs*
iu-gen-matching-fastq-files*
iu-interleave-fastq*
iu-merge-pairs*
iu-remove-ids-from-fastq*
iu-subsample-fastq*
iu-trim-fastq*
iu-trim-V6-primers*
iu-visualize-mismatch-distribution*
iu-visualize-plot-dicts*
iu-visualize-qual-dicts*
jsonschema*
__pycache__/
sqlformat*
vortex1$ 
```

The extensive test suite, for example, can be run with:

```
vortex1$ anvi-self-test --suite mini
[...]
vortex1$ 

```
